onnx/test/optimizer_test.py

```diff
@@ -19,6 +19,33 @@ def _optimized(self, graph, opts):
         checker.check_model(optimized_model)
         return optimized_model
 
+    def test_identity(self):
+        identity = helper.make_node("Identity", ["X"], ["Y"])
+        graph = helper.make_graph(
+            [identity],
+            "test",
+            [helper.make_tensor_value_info("X", TensorProto.FLOAT, (5,))],
+            [helper.make_tensor_value_info("Y", TensorProto.FLOAT, (5,))])
+        optimized_model = self._optimized(graph, ["eliminate_identity"])
+
```

Comment:
check len(node) == 0 and also check the output is "Y"

Reply:
@houseroad 
The output is "X" because Identity is eliminated also "Y".

Reply:
We probably need to keep output name if the considered Identity is a leaf node.

Reply:
@wschin You mean case that Identity is an output node right? Agree.

Reply:
@fumihwh yeah, I mean, check the output is "X" in the graph.